google/auth/impersonated_credentials.py

```diff
@@ -271,13 +285,19 @@ def _update_token(self, request):
             principal=self._target_principal,
             headers=headers,
             body=body,
+            universe_domain=self.universe_domain,
             iam_endpoint_override=self._iam_endpoint_override,
         )
 
+    def get_iam_sign_endpoint(self):
```

Comment:
If this method was introduced for testing only, please remove it and use monkeypatch for testing like rest of the repo.

Reply:
removed